### PR TITLE
provider/kubernetes: Retrieve credentials for server group

### PR DESCRIPTION
**Pull request by lwander**
The `credentials` field is needed for clone server group operations to resolve valid images for the given namespace. 

@duftler

### CODE REVIEW

File: clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesServerGroup.groovy [hunk 1]
```diff
@@ -30,6 +30,7 @@ class KubernetesServerGroup implements ServerGroup, Serializable {
   String name
   String type = "kubernetes"
   String region
+  String credentials
```
Comment (duftler) [inline clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesServerGroup.groovy]: Wouldn't using `account` be more consistent?

**Comment by duftler** [comment]
Looks like the unit tests need to be updated to reflect the modified constructor.

**Comment by lwander** [comment]
I changed every use of `credentials` when referring to an account name to `account`. `credentials` is still used to refer to the class containing the authentication details & client.

**Comment by duftler** [comment]
Looks like Metal Travis has now decided to not even try.
LGTM, once you can get Metal Travis on board.

**Pull request merged**